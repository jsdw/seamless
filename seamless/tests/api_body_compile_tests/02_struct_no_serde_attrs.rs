#[seamless::ApiBody]
struct Foo {
    /// Hello there!
    hello: usize,
    /// Barry!
    #[serde(rename = "foo")]
    bar: String
}

fn main () {

}