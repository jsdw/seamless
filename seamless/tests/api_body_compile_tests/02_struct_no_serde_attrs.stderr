error: serde attributes not allowed; ApiBody macro handles that
 --> $DIR/02_struct_no_serde_attrs.rs:6:5
  |
6 |     #[serde(rename = "foo")]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^
